manim/mobject/svg/code_mobject.py

```diff
@@ -263,9 +263,7 @@ def ensure_valid_file(self):
             if os.path.exists(path):
                 self.file_path = path
                 return
-        error = "From: {}, could not find {} at either of these locations: {}".format(
-            os.getcwd(), self.file_name, possible_paths
-        )
+        error = f"From: {os.getcwd()}, could not find {self.file_name} at either of these locations: {possible_paths}"
```

Comment:
This line seems a little long, but if black doesn't complain it should still be < 90 chars. (Could you please check, and if it is longer, break it up?)

Reply:
Funny... Black did not change it (neither locally nor in the repository), but the line was clearly longer than 90 chars.